blazor/datagrid/batch-editing.md

```diff
@@ -1037,4 +1037,17 @@ public class OrderDetails
 {% endhighlight %}
 {% endtabs %}
 
-{% previewsample "https://blazorplayground.syncfusion.com/embed/hjrStoqXKmIboOTG?appbar=false&editor=false&result=true&errorlist=false&theme=bootstrap5" %}
\ No newline at end of file
+{% previewsample "https://blazorplayground.syncfusion.com/embed/hjrStoqXKmIboOTG?appbar=false&editor=false&result=true&errorlist=false&theme=bootstrap5" %}
+
+## Supported events for batch editing
+
+The Syncfusion Blazor DataGrid triggers the following events during `Batch` editing:
```

Comment:
Batch editing in the Syncfusion Blazor DataGrid offers a powerful way to edit multiple records simultaneously. Understanding the sequence and purpose of events triggered during this process enables users to customize or extend the data grid's functionality. This section outlines the key events associated with batch editing, providing essential information to harness the full potential of this feature. ----- Got this from AI